vmm/src/memory_manager.rs

```diff
@@ -126,6 +127,8 @@ struct GuestRamMapping {
     size: u64,
     #[allow(dead_code)]
     zone_id: String,
+    #[allow(dead_code)]
```

Comment:
Why is this dead code?

Reply:
Oh just because it's not used at the point where this commit introduces this new field. But I remove the `dead_code` as soon as it's being used and that `clippy` doesn't complain anymore.